### ISSUE TITLE
Failure of runtest's 171 testcase

### ISSUE THREAD

**Issue by yanghyeimm** [namhyung/uftrace#241]
I did all test cases of runtest 171, and I got to know all of these were failed.

```
test command: ../uftrace --no-pager -L.. -S script.py t-openclose
=========== original ===========
# DURATION    TID     FUNCTION
            [17587] | main() {
   5.136 us [17587] |   fopen("/dev/null");
   1.700 us [17587] |   fclose();
   9.381 us [17587] | } /* main */

===========  result  ===========
# DURATION    TID     FUNCTION
=========== expected ===========
fopen(/dev/null)
171 script_option       : NG NG NG NG NG NG NG NG NG NG
```

I did run command but I got the same result which I expected.

```
$ ../uftrace --no-pager -L.. -S testscript.py t-openclose
# DURATION    TID     FUNCTION
            [ 3194] | main() {
  24.289 us [ 3194] |   fopen("/dev/null");
   2.433 us [ 3194] |   fclose();
  29.997 us [ 3194] | } /* main */
```
I am continuely getting 'NG'.
Could you explain me this problem?

**Comment by namhyung** [comment]
Did you install python development package like `libpython2.7-dev`?  Please check you have `check-deps/have_libpython2.7` file.

**Comment by yanghyeimm** [comment]
It exists in my local..

**Comment by yanghyeimm** [comment]
oh sorry, i checked again, and i did not find it.

**Comment by honggyukim** [comment]
If so, you have to first install `libpython2.7-dev` then must run `configure` script again to check the library.

**Comment by yanghyeimm** [comment]
I checked many times what you said. I installed `libpython2.7-dev` and ran `configure` script. 
And I also checked `have_libpython2.7` in `check-deps` directory by running `sudo make` command.
but I got same failures again...
Actually I used `sudo make` command instead of using only `make` command because I couldn't run `make` command without `sudo`.
Is it a matter?

**Comment by namhyung** [comment]
I don't think it matters but generally running make with sudo is not a good practice.  I suggest you to delete all files and then build without sudo again.

**Comment by honggyukim** [comment]
I also think that you better delete your current working directory and download uftrace in a clean space without sudo. sudo is needed only for `make install` if prefix is not changed.

**Comment by yanghyeimm** [comment]
Thank you. I'll try.

**Comment by namhyung** [comment]
Any updates?
